### ISSUE TITLE
many missing database names for common names in Prey.csv

### ISSUE THREAD

**Issue by jhpoelen** [GoMexSI/abascal#5]
In order for name resolving to work, best practice is to use appropriate scientific names, rather than common names. Common names are often confusing and lead to data errors. 

Suggest to populate Prey.csv columns "PREY_DATABASE_NAME" with scientific name, if none is provided in the source.

**Comment by jhpoelen** [comment]
@tmitchell1 @jsimons9 @iabascal please let me know if you'd like to schedule a meeting to review this valuable dataset so that the data can be integrated into the existing corpus of interactions.

**Comment by jsimons9** [comment]
Deleted personal email.

**Comment by tmitchell1** [comment]
@jhpoelen @jsimons9 @iabascal - I very quickly fixed the casing in the References sheet. I also filled in blank/unused cells with "NA"

**Comment by jhpoelen** [comment]
@tmitchell1 thanks for the quick reply. Did you also manage to provide the Scientific names for Prey.csv column PREY_DATABASE_NAME as suggested in this issue? 

Happy to help provide more info if needed.

**Comment by tmitchell1** [comment]
@jhpoelen No, I didn't.  I just went through the References real quick. Most, if not all, of Michelle's data was uploaded without the spatial data.

**Comment by jhpoelen** [comment]
@tmitchell1 thanks for letting me know. Please note that ref id in other files (e.g. Prey.csv) also need capitalization. Also, please note #4 - referenced, but undefined reference ids.
